windows/windows_update_install/install_msu_file.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Set the .msu file installation related parameters"
+  ansible.builtin.set_fact:
+    msu_kb_number: "{{ msu_file_name | regex_search('(?i)KB\\d+') | upper }}"
+    msu_file_dest_path: "{{ msu_dir_path }}\\{{msu_file_name }}"
+    msu_install_timeout: 7200
+    msu_become_user: "{{ 'SYSTEM' if guest_os_ansible_architecture == '32-bit' else 'Administrator' }}"
+
+- name: "Enable the user Administrator"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "net user Administrator /active:yes"
+  when: vm_username | lower != "administrator"
+
+- name: "Install the .msu file"
+  ansible.windows.win_shell: >-
+    wusa.exe {{ msu_file_dest_path }} /quiet /norestart
+  delegate_to: "{{ vm_guest_ip }}"
+  register: install_msu_result
+  become: true
+  become_method: runas
+  become_user: "{{ msu_become_user }}"
+  async: "{{ msu_install_timeout }}"
+  poll: 0
+  environment:
+    ANSIBLE_WIN_ASYNC_STARTUP_TIMEOUT: 10
+
+- name: "Check if the .msu file is installed before restart"
```

Comment:
`install_msu_result.ansible_job_id` is not checked to make sure Async task is started successfully.